stdlib/number.gr

```diff
@@ -49,7 +49,7 @@ export let div = (/)
 @disableGC
 export let sqrt = (x: Number) => {
   let x = WasmI32.fromGrain(x)
-  let xval = coerceFloat64(x)
+  let xval = coerceNumberToWasmF64(WasmI32.toGrain(x) : Number)
```

Comment:
Is there a reason the `x` is shadowed here and then converted back into a Grain number?

Reply:
Yeah, in this case we could swap this line with the line above to avoid the need to cast back to `Number`.

Reply:
Swapped the lines and removed the cast.